ui/admin-portal/src/app/components/candidates/view/occupation/experience/view-candidate-job-experience.component.html

```diff
@@ -63,7 +63,7 @@
 
         <div class="col-10">
           <h6 class="card-title">{{experience.role}}</h6>
-          <p *ngIf="experience.description">{{experience.description}}</p>
+          <p *ngIf="experience.description" [ngClass]="isHtml(experience?.description)?'html':'text'" [innerHtml]="experience.description"></p>
```

Comment:
wonder if there's any harm defaulting to 'html' in all cases here?

Reply:
I found this issue in several places, hence the repeated code in the public, admin and candidate portals. Maybe the answer is to convert all the textual descriptions to HTML description as part of an upgrade then we don't need to worry about the two formats (text, and html). It would certainly reduce the amount of change and also reduce the complexity of dealing with both, which might get messier over time. Let's chat about this when we zoom.

Reply:
thanks so much Tim - look forward to chatting with you soon!